src/components/OptionsSelector.js

```diff
@@ -203,21 +215,26 @@ class OptionsSelector extends Component {
                             || this.props.translate('optionsSelector.nameEmailOrPhoneNumber')}
                     />
                 </View>
-                <OptionsList
-                    ref={el => this.list = el}
-                    optionHoveredStyle={styles.hoveredComponentBG}
-                    onSelectRow={this.selectRow}
-                    sections={this.props.sections}
-                    focusedIndex={this.state.focusedIndex}
-                    selectedOptions={this.props.selectedOptions}
-                    canSelectMultipleOptions={this.props.canSelectMultipleOptions}
-                    hideSectionHeaders={this.props.hideSectionHeaders}
-                    headerMessage={this.props.headerMessage}
-                    disableFocusOptions={this.props.disableArrowKeysActions}
-                    hideAdditionalOptionStates={this.props.hideAdditionalOptionStates}
-                    forceTextUnreadStyle={this.props.forceTextUnreadStyle}
-                    showTitleTooltip={this.props.showTitleTooltip}
-                />
+                <View style={[styles.flex1, styles.w100, styles.pRelative]}>
+                    {!this.props.isLoading ? (
+                        <OptionsList
+                            ref={el => this.list = el}
+                            optionHoveredStyle={styles.hoveredComponentBG}
+                            onSelectRow={this.selectRow}
+                            sections={this.props.sections}
+                            focusedIndex={this.state.focusedIndex}
+                            selectedOptions={this.props.selectedOptions}
+                            canSelectMultipleOptions={this.props.canSelectMultipleOptions}
+                            hideSectionHeaders={this.props.hideSectionHeaders}
+                            headerMessage={this.props.headerMessage}
+                            disableFocusOptions={this.props.disableArrowKeysActions}
+                            hideAdditionalOptionStates={this.props.hideAdditionalOptionStates}
+                            forceTextUnreadStyle={this.props.forceTextUnreadStyle}
+                            showTitleTooltip={this.props.showTitleTooltip}
+                        />
+                    ) : <FullScreenLoadingIndicator isVisible={this.props.isLoading} />}
```

Comment:
Why do we have `isVisible={this.props.isLoading}`? We don't render the `<FullScreenLoadingIndicator/>` unless we are loading right?

Reply:
Sorry my bad, it's unnecessary, will remove it.

Reply:
Please do that then.